food-truck-frontend/src/components/util/HiddenAttributeForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { TextField } from "@material-ui/core";
+import React from "react";
+import Form, { Props as FormProps } from "../Form";
+
+export interface HiddenEntry {
+  name: string;
+  defaultValue: any;
+}
+
+export interface HiddenAttributeFormProps {
+  hiddenAttrs: HiddenEntry[];
+}
+
+class HiddenField extends React.Component<HiddenEntry> {
+  constructor(props: HiddenEntry) {
+    super(props);
+  }
+  render() {
+    return <div />;
+  }
+}
+
+function HiddenAttributeForm(props: HiddenAttributeFormProps & FormProps) {
+  return (
+    <Form {...props}>
+      {React.Children.map(props.children, c => c)}
+      {props.hiddenAttrs.map((tpl) => (
+        <HiddenField key={`hidden${tpl.name}`} {...tpl} />
+      ))}
+    </Form>
+  );
+}
+
+export default HiddenAttributeForm;
```

Comment:
This looks like it will work, but we could also just use CSS "display: none"

Reply:
Huh. Learn something new every day. Yeah, I could've used that. Well, it's working as-is, so I won't change it unless we decide that CSS property would work better.